cinema-frontend/package-lock.json

```diff
@@ -8212,6 +8212,11 @@
       "integrity": "sha1-yobR/ogoFpsBICCOPchCS524NCw=",
       "dev": true
     },
+    "ngx-bootstrap": {
```

Comment:
IMO `package-lock.json` should be in .gitignore

Reply:
Please check this

Reply:
This file must to be in git